oe_theme.theme

```diff
@@ -1592,6 +1594,9 @@ function oe_theme_preprocess_page(array &$variables): void {
     'class' => ['ecl-u-pb-xl'],
     'id' => ['main-content'],
   ]);
+  // Put 'data-ecl-has-menu' attribute only if we have a menu inside
+  // the site header.
+  $variables['ecl_has_menu_attribute'] = !empty($variables['page']['navigation']) ? 'data-ecl-has-menu' : '';
```

Comment:
This should not be needed either after EWPP-1250 is merged. The attribute should is added by ecl only if there are menu items.